### ISSUE TITLE
Can Trimmomatic be skipped while running Wochenende ?

### ISSUE THREAD

**Issue by arpit20328** [MHH-RCUG/Wochenende#223]
Hi 

So after all settings in config.yml 

I ran 

 python3 run_Wochenende.py  arpit_R1.fastq --aligner bwamem --readType PE --metagenome 2021_12_meta_fungi_human_masked  --threads 144  --no_abra --remove_mismatching 3 --no_fastqc

can I stop Trimmomatic ? I have already removed adapters etc. 

**Please note I am not using nf or slurm.  I am just executing python scripts.**

**Comment by colindaven** [comment]
Hi Arpit, I don't think you can turn off trimmomatic. We always ran it from raw sequence to avoid saving the trimmed.

I can also advise you to use max 24 threads for performance reasons (there are minimal gains after 20-24 threads for almost all tools). If you're lucky enough to have a big machine, just start six or seven python processes in parallel using this  system, or multiple shells.

python3 run_Wochenende.py args &
python3 run_Wochenende.py args &
python3 run_Wochenende.py args &
python3 run_Wochenende.py args &

**Comment by colindaven** [comment]
That said, you can also comment out Lines 1541 -48 in the python script and see if that works for you (assuming paired end PE reads, there is a different section for single ended SE reads).

https://github.com/MHH-RCUG/Wochenende/blob/0eed8dafe905ded5fd0f77a8c7863e90f173d2d2/run_Wochenende.py#L1541

**Comment by arpit20328** [comment]
@colindaven will try

**Comment by arpit20328** [comment]
@colindaven yes trimming stopped after commenting 1541 to 1548 lines . thanks
